spec/peep_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'peep'
+
+describe Peep do
+  describe '.create' do
+    it 'saves the peep in the database' do
+      peep = 'This is the first peep'
```

Comment:
Clean testing here. Maybe let's look at how you can use doubles to untie the db relationship.